tests/unit_tests/bbox_formats_test.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+
+import numpy as np
+import torch
+
+from super_gradients.training.utils.bbox_formats import (
+    CXCYWHCoordinateFormat,
+    NormalizedXYXYCoordinateFormat,
+    NormalizedXYWHCoordinateFormat,
+    XYWHCoordinateFormat,
+    YXYXCoordinateFormat,
+    XYXYCoordinateFormat,
+    NormalizedCXCYWHCoordinateFormat,
+    convert_bboxes,
+)
+
+
+class BBoxFormatsTest(unittest.TestCase):
```

Comment:
Nice tests.
I think it'd be also a good idea to take some fixed boxes conversion cases and test them as well, as I'm worried that these might bot cover the correctness of the conversion mechanism itself.